app_config.py

```diff
@@ -0,0 +1,5 @@
+AUTHORITY = "https://login.microsoftonline.com/Enter_the_Tenant_Name_Here"
+CLIENT_ID = "Enter_the_Application_Id_here"
+CLIENT_SECRET = "Enter_the_Client_Secret_Here"
+SCOPE = ["https://graph.microsoft.com/User.Read"]
```

Comment:
This does a cache miss as we don't handle this in the library

Reply:
I believe that would also be the case for all MSALs. No?
On the other hand, is there an official document somewhere mentioning all those canonical scope names? If so, we can reference to it.

Reply:
I had a PR in MSAL Python review which is put on hold solving this. Might have to refactor it as lot of code changes have happened since then : https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/26. We can make this change once we confirm with the official documentation mentioning it.